app/utils/sentry.js

```diff
@@ -0,0 +1,32 @@
+const Sentry = require('@sentry/node')
+
+function initialiseSentry () {
+  Sentry.init({
+    dsn: process.env.SENTRY_DSN,
+    environment: process.env.ENVIRONMENT,
+    beforeSend (event) {
+      if (event.request) {
+        delete event.request // This can include sensitive data such as card numbers
+      }
+      return event
+    }
+  })
+  return Sentry
+}
+
+const addSentryToErrorLevel = (originalLogger) => {
```

Comment:
total nitpick, but no need for brackets around `(originalLogger)`

Reply:
nitpick away 👍fixed